rpcd/playbooks/maas-20151013-2-pre-migrate.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Part of MaaS Migration: maas-20151013.yml
+- name: Record Prior Checks before MaaS Migration
+  hosts: hosts
```

Comment:
Perhaps we could change this to hosts[0] since all the tasks in here run against hosts[0] - it would make it simpler because the final task will fail on the other hosts since it tries to evaluate the with_items before the when clause (since with_items var doesn't exist on the other hosts it will fail out on those nodes - which isn't super important but isn't tidy either).